src/site/includes/education-sco.html

```diff
@@ -48,7 +48,7 @@ <h1>
         {% endif %}
 
         {% if social != empty %}
-          {% include "src/site/components/merger-social.html" %}
+          {% include "src/site/components/merger-social-sco.html" %}
```

Comment:
@ncksllvn just an fyi- i'm not sure why we wouldn't want this to be applied across the site if it better meets the design standards.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/blob/05e109dcf005b9c30e47dad1cd13617209185472/src/site/components/merger-social-sco.html#L45
https://github.com/department-of-veterans-affairs/vets-website/blob/05e109dcf005b9c30e47dad1cd13617209185472/src/site/components/merger-social-sco.html#L50

These 2 lines are specific to https://github.com/department-of-veterans-affairs/vagov-content/blob/master/pages/education/school-resources.md

Reply:
thanks @zurbergram